spec/factories/works.rb

```diff
@@ -4,7 +4,7 @@
   factory :work do
     sequence(:title) { |n| "Work #{n}" }
     user
-    association :collection, :with_druid # rubocop:disable FactoryBot/AssociationStyle
+    collection factory: %i[collection with_druid]
```

Comment:
I couldn't figure this one out!

Reply:
The change was not obvious! I read the documentation three times before trying this on a whim and whaddya know?